rke/k8s_rke_system_images.go

```diff
@@ -2654,37 +2654,37 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
-			DNSmasq:                   m("gcr.io/google_containers/k8s-dns-dnsmasq-nanny:1.15.0"),
-			KubeDNSSidecar:            m("gcr.io/google_containers/k8s-dns-sidecar:1.15.0"),
-			KubeDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.7.1"),
-			Flannel:                   m("quay.io/coreos/flannel:v0.12.0"),
+			KubeDNS:                   m("rancher/k8s-dns-kube-dns:1.15.0"),
+			DNSmasq:                   m("rancher/k8s-dns-dnsmasq-nanny:1.15.0"),
+			KubeDNSSidecar:            m("rancher/k8s-dns-sidecar:1.15.0"),
+			KubeDNSAutoscaler:         m("rancher/cluster-proportional-autoscaler:1.7.1"),
+			Flannel:                   m("rancher/coreos-flannel:v0.12.0"),
 			FlannelCNI:                m("rancher/flannel-cni:v0.3.0-rancher6"),
-			CalicoNode:                m("quay.io/calico/node:v3.13.4"),
-			CalicoCNI:                 m("quay.io/calico/cni:v3.13.4"),
-			CalicoControllers:         m("quay.io/calico/kube-controllers:v3.13.4"),
-			CalicoCtl:                 m("quay.io/calico/ctl:v3.13.4"),
-			CalicoFlexVol:             m("quay.io/calico/pod2daemon-flexvol:v3.13.4"),
-			CanalNode:                 m("quay.io/calico/node:v3.13.4"),
-			CanalCNI:                  m("quay.io/calico/cni:v3.13.4"),
-			CanalFlannel:              m("quay.io/coreos/flannel:v0.12.0"),
-			CanalFlexVol:              m("quay.io/calico/pod2daemon-flexvol:v3.13.4"),
+			CalicoNode:                m("rancher/calico-node:v3.13.4"),
+			CalicoCNI:                 m("rancher/calico-cni:v3.13.4"),
+			CalicoControllers:         m("rancher/calico-kube-controllers:v3.13.4"),
+			CalicoCtl:                 m("rancher/calico-ctl:v3.13.4"),
+			CalicoFlexVol:             m("rancher/calico-pod2daemon-flexvol:v3.13.4"),
+			CanalNode:                 m("rancher/calico-node:v3.13.4"),
+			CanalCNI:                  m("rancher/calico-cni:v3.13.4"),
+			CanalFlannel:              m("rancher/coreos-flannel:v0.12.0"),
+			CanalFlexVol:              m("rancher/calico-pod2daemon-flexvol:v3.13.4"),
 			WeaveNode:                 m("weaveworks/weave-kube:2.6.4"),
 			WeaveCNI:                  m("weaveworks/weave-npc:2.6.4"),
-			PodInfraContainer:         m("gcr.io/google_containers/pause:3.1"),
+			PodInfraContainer:         m("rancher/pause:3.1"),
 			Ingress:                   m("rancher/nginx-ingress-controller:nginx-0.32.0-rancher1"),
-			IngressBackend:            m("k8s.gcr.io/defaultbackend:1.5-rancher1"),
-			MetricsServer:             m("gcr.io/google_containers/metrics-server:v0.3.4"),
-			CoreDNS:                   m("coredns/coredns:1.6.2"),
-			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.7.1"),
+			IngressBackend:            m("rancher/nginx-ingress-controller-defaultbackend:1.5-rancher1"),
+			MetricsServer:             m("rancher/metrics-server:v0.3.4"),
+			CoreDNS:                   m("rancher/coredns:1.6.2"),
```

Comment:
This is not the right image: https://hub.docker.com/repository/docker/rancher/coredns/tags?page=1

This one shouldn't be changed.